lua/entities/rail_shell/init.lua

```diff
@@ -38,7 +40,7 @@ function ENT:Think()
 
     if self.AirburstTime < CurTime() then
         local owner = IsValid( self:GetOwner() ) and self:GetOwner()
-        local inflictor = owner or self.Turret
+        local inflictor = IsValid( owner ) and owner or IsValid( self.Turret ) and self.Turret or self

```

Comment:
Perhaps some parenthesis to make this a bit more readable?

Reply:
not sure how you'd do that, it will get ugly even with parenthesis as far as i know

Reply:
```suggestion
        local inflictor = IsValid( owner ) and owner or ( IsValid( self.Turret ) and self.Turret or self )

```

Reply:
returns nil

Reply:
oh wait no, but the self is still in a weird spot there